testsuite/tests/apicast/parameters/test_apicast_pagination.py

```diff
@@ -0,0 +1,48 @@
+"""Test for apicast pagination"""
+import asyncio
+from concurrent.futures.thread import ThreadPoolExecutor
+
+import pytest
+
+from testsuite.utils import blame
+
+
+@pytest.fixture(scope="module")
+def event_loop():
+    """Event loop for creation of services"""
+    with ThreadPoolExecutor() as pool:
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        loop.set_default_executor(pool)
+        yield loop
+        loop.close()
+
+
+# pylint: disable=too-many-arguments
+@pytest.fixture(scope="module")
+async def many_services(request, custom_service, service_proxy_settings, lifecycle_hooks, custom_backend, event_loop):
+    """Creation of 500+ services"""
+    backend_mapping = {"/": custom_backend("backend")}
+
+    def _create_services():
+        custom_service({"name": blame(request, "svc")}, service_proxy_settings, backend_mapping, hooks=lifecycle_hooks)
+
+    return await asyncio.gather(
+        *(event_loop.run_in_executor(None, _create_services) for _ in range(505))
+    )
+
+
+# pylint: disable=unused-argument
+@pytest.mark.issue("https://issues.redhat.com/browse/THREESCALE-8373")
+def test(many_services, api_client, staging_gateway):
```

Comment:
`test_apicast_pagination` please. You may do the review in reportportal one day, then you will understand

Reply:
at least `@pytest.mark.sandbag`